README.md

```diff
@@ -98,26 +101,40 @@ The following command will install most of the required dependencies:
 
     sudo apt-get install libboost-program-options-dev libnuma-dev numactl libhwloc-dev moreutils linux-tools-common linux-tools-generic libtbb-dev
 
-Building MT-KaHyPar
+Building Mt-KaHyPar
 -----------
 
 1. Clone the repository including submodules:
 
    ```git clone --depth=1 --recursive git@github.com:kittobi1992/mt-kahypar.git```
 2. Create a build directory: `mkdir build && cd build`
 3. Run cmake: `cmake .. -DCMAKE_BUILD_TYPE=RELEASE`
-4. Run make: `make KaHyPar`
+4. Run make: `make MtKaHyPar -j8`
```

Comment:
why tell people how many threads to use? I think make -j gives you max parallelism of the machine.